app/code/Magento/CatalogInventory/Model/Plugin/AddStockItemsProducts.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ *
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\CatalogInventory\Model\Plugin;
+
+/**
+ * Stock Item as a part of ExtensionAttributes
```

Comment:
Please mark functionality as deprecated (in the same way it happens here \Magento\CatalogInventory\Model\Plugin\AfterProductLoad).

Reply:
Hi, I marked functionality as deprecated